app/router/auth.py

```diff
@@ -1,6 +1,6 @@
 from fastapi import APIRouter, Depends
```

Comment:
the file `app/router/__init_.py` is wrongly named. There should be two underscores not one